src/clear_map.h

```diff
@@ -269,6 +269,7 @@ static inline void MakeClear(TileIndex t, ClearGround g, uint density)
 	SetClearGroundDensity(t, g, density); // Sets m5
 	_me[t].m6 = 0;
 	_me[t].m7 = 0;
+	_me[t].m8 = 0;
```

Comment:
This line is needed as calling DoClearSquare(tile) needs it to remove railway type of tunnelbridges.